src/Library/demos/Progress Bar/main.blp

```diff
@@ -0,0 +1,43 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "Progress Bar";
+  description: _("Display long running operation");
+
+  Box {
+    orientation: vertical;
+    halign: center;
+
+      ProgressBar first {
+        fraction: 0.2;
+        show-text: true;
+        margin-bottom: 24;
+      }
+
+      ProgressBar second {
+        inverted: true;
+        pulse-step: 0.25;
+        show-text: true;
+        text: "";
+        margin-bottom: 24;
+      }
```

Comment:
I like the idea of having a second progress bar tracking the same value in a different way. I think the demonstration could be a little more intuitive, such as a "1/4 parts complete" or "time remaining" countdown.

Reply:
Gotcha , I'll implement it